packages/map-template/src/components/LocationDetails/LocationDetails.scss

```diff
@@ -6,10 +6,11 @@
     display: grid;
     height: 100%;
     overflow-y: hidden;
-    grid-template-rows: 40px 1fr 64px;
+    grid-template-rows: min-content 1fr 64px;
```

Comment:
Did you test this change on a Location with a lot of text? I'm always worried when we change styles here 😅

Reply:
Yes, it has been tested. The UX department had an adjustment for the `mi-location-info` component, to only allow max 3 lines of text, then show `...` if the text is longer. I have pushed the latest changes now. @ammapspeople

Reply:
@andreeaceachir142 I believe the feature with 3 lines was already there 🤔

Reply:
Ah, wrong place. Never mind.